### PR TITLE
Added native SSL support

### PR DESCRIPTION
**Pull request by hbs**

### CODE REVIEW

File: warp10/src/main/java/io/warp10/standalone/Warp.java [hunk 1]
```diff
@@ -262,28 +267,50 @@ public static void main(String[] args) throws Exception {
...
+    if (useHTTPS) {
+      httpsConnector = SSLUtils.getConnector(server, Configuration.STANDALONE_PREFIX);
+      httpsConnector.setName("Warp 10 Standalone HTTPS Endpoint");      
+      connectors.add(httpsConnector);
     }
     
-    connector.setName("Continuum Standalone Egress");
-    
-    server.setConnectors(new Connector[] { connector });
+    server.setConnectors(connectors.toArray(new Connector[0]));
```
Comment (ftence) [inline warp10/src/main/java/io/warp10/standalone/Warp.java]: `.toArray(new Connector[connectors.size()])`

File: warp10/src/main/java/io/warp10/standalone/Warp.java [hunk 2]
```diff
@@ -441,7 +468,9 @@ public static void main(String[] args) throws Exception {
     }
```
Comment (ftence) [inline warp10/src/main/java/io/warp10/standalone/Warp.java]: Display endpoint for HTTP or HTTPS in the try block just above.

File: warp10/src/main/java/io/warp10/continuum/egress/Egress.java [hunk 3]
```diff
@@ -109,15 +135,8 @@ public Egress(KeyStore keystore, Properties props, boolean fetcher) throws Excep
...
-    server = new Server();
-    ServerConnector connector = new ServerConnector(server, acceptors, selectors);
-    connector.setIdleTimeout(idleTimeout);
-    connector.setPort(port);
-    connector.setHost(host);
-    connector.setName("Continuum Egress");
-    
-    server.setConnectors(new Connector[] { connector });
+                
+    server.setConnectors(connectors.toArray(new Connector[0]));
```
Comment (ftence) [inline warp10/src/main/java/io/warp10/continuum/egress/Egress.java]: `.toArray(new Connector[connectors.size()])` would be nicer.

File: warp10/src/main/java/io/warp10/continuum/ingress/Ingress.java [hunk 4]
```diff
@@ -526,14 +521,36 @@ public void run() {
       queue = new BlockingArrayQueue<Runnable>(queuesize);
     }
     
-    Server server = new Server(new QueuedThreadPool(maxThreads,8, (int) idleTimeout, queue));
-    ServerConnector connector = new ServerConnector(server, acceptors, selectors);
-    connector.setIdleTimeout(idleTimeout);
-    connector.setPort(port);
-    connector.setHost(host);
-    connector.setName("Continuum Ingress");
+    Server server = new Server(new QueuedThreadPool(maxThreads,8, 60000, queue));
```
[the remark this answers is not in the document]
Comment (hbs) [reply warp10/src/main/java/io/warp10/continuum/ingress/Ingress.java]: The connector timeout is still set as before, the timeout for the queue was incorrectly set to the value set for the connector idle timeout.

**Comment by hbs** [comment]
Please Take Another Look

**Comment by ftence** [comment]
Another look taken, standalone ingress, egress and http plugin tested successfully, I think you're good to go.

**Pull request merged**